src/droplet.F

```diff
@@ -413,6 +415,12 @@ subroutine droplet_driver(dt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,mf,zs,
                                  xh,xf,yh,yf,zh,zf,zs,znt,sigma,sigmaf, &
                                  ua,va,wa,ta,qa,rho,prs,sigdot)
 
+      !Temporary for testing purposes:
+      rhval = (ru*tval*rhoval*qval)/(mw*eslf(prsval,tval))
+      if (rhval .lt. 0.3 .or. rhval .gt. 1.2) then
+      write(*,'(a8,i,8e15.6)') 'DHR3',np,rhval,tval,rhoval,qval,x3d,y3d,z3d,pdata(np,prrp)
+      end if
```

Comment:
Do we have to keep this `write` statement? The `write` function is not supported on the GPU.

Reply:
Could you just comment it out for now? We won't need it in the long run, but it's been helpful for setting up the case

Reply:
Ok, that sounds good. I will just comment it out for now.